rust/rsnano/src/voting/inactive_cache_information.rs

```diff
@@ -0,0 +1,45 @@
+use crate::core::{Account, BlockHash};
+use crate::voting::InactiveCacheStatus;
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+
+/// Information on the status of inactive cache information
+#[derive(Clone)]
+pub struct InactiveCacheInformation {
+    pub arrival: SystemTime,
+    pub hash: BlockHash,
+    pub status: InactiveCacheStatus,
+    pub voters: Vec<(Account, SystemTime)>,
```

Comment:
I really like that you don't wanted to use 64 but a type that expresses what this value is. However should this be SystemTime or Instant? For simplicity maybe a u64 is currently the best solution until we don't have to pass value back that to C++